backend/app/celery/resource_monitor.py

```
@@ -0,0 +1,194 @@
+            f"Check interval: {self.check_interval}s, "
+            f"Monitoring queues: {', '.join(self.queue_names)}"
+        )
+
+        while not self._should_stop:
+            try:
+                cpu = self.get_cpu_usage()
+                memory = self.get_memory_usage()
+                should_pause_now = self.should_pause(cpu, memory)
+                # Remove this line later
+                logger.info(
+                    "Memmory Usage: {:.2f}%, CPU Usage: {:.2f}%".format(memory, cpu)
```

Ruff (0.14.7)

Use f-string instead of `format` call

Convert to f-string

(UP032)